tungsten_tempest_plugin/tests/api/contrail/test_config_client.py

```diff
@@ -195,10 +200,12 @@ def test_list_global_system_configs(self):
         with self.rbac_utils.override_role(self):
             self.config_client.list_global_system_configs()
 
+    @testtools.skipUnless(float(CONF.sdn.contrail_version) < 5,
+                          'Not supported in OpenContrail versions < 5')
```

Comment:
I think the logic of the message may be backwards. skipUnless means this test will run for versions less than 5 but not for 5+, so presumably the message should be 'Not supported in OpenContrail version >= 5'

Reply:
Hemachandra Reddy confirmed that this test should be run agains version < 5, and skipped for >= 5. So the message needs to be updated, not condition

Reply:
Yes, that's what I meant. I agree the test condition is correct, the message is what is backwards.